Porting.md

```diff
@@ -34,19 +34,34 @@ chmod a+x ~/bin/repo
 
 ```
 mkdir halium && cd halium
-repo init -u https://github.com/Halium/android -b halium-5.1
-# or
+```
+
+If the target device has Android 7.1 or LineageOS 14.1 support, it's recommended to select halium-7.1
+```
 repo init -u https://github.com/Halium/android -b halium-7.1
 ```
 
+If your device does not have Android 7.1 or LineageOS 14.1 support but has Android 5.1 or CyanogenMod 12.1 select halium-5.1
+
+```
+repo init -u https://github.com/Halium/android -b halium-5.1
+```
+
 **//TODO: Add notes about halium-5.1 and halium-7.1**
 
+halium-7.1 is based on LineageOS 14.1
+halium-5.1 is based on CyanogenMod 12.1
+
 #### Step 4: Download the source code
 
 ```
-repo sync
+repo sync -c -j10
```

Comment:
I wouldn't add them here but in the manifest as default (if they aren't already)